src/gui/tray/syncstatussummary.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include "accountstate.h"
+#include "folderman.h"
+
+#include <theme.h>
+#include <folder.h>
+
+#include <QObject>
+
+namespace OCC {
+
+class SyncStatusSummary : public QObject
+{
+    Q_OBJECT
+
+    Q_PROPERTY(double syncProgress READ syncProgress NOTIFY syncProgressChanged)
+    Q_PROPERTY(QUrl syncIcon READ syncIcon NOTIFY syncIconChanged)
+    Q_PROPERTY(bool syncing READ syncing NOTIFY syncingChanged)
+    Q_PROPERTY(QString syncStatusString READ syncStatusString NOTIFY syncStatusStringChanged)
+    Q_PROPERTY(QString syncStatusDetailString READ syncStatusDetailString NOTIFY syncStatusDetailStringChanged)
+
+public:
+    explicit SyncStatusSummary(QObject *parent = nullptr);
+
+    Q_INVOKABLE void load();
+
+    double syncProgress() const;
+    QUrl syncIcon() const;
+    bool syncing() const;
+    QString syncString() const;
```

Comment:
why is there no property for that even if there is apparently a getter and a notify signal ?